### DISCUSSION TITLE
Limiting the number of players in a cell

### DISCUSSION THREAD

**Discussion by Deimosssss** [Zrips/CMI#7178] [Enhancement]
Hello everyone,

My idea is that it would be very useful when it will be there possibility to setting, how many players could be jailed in specific cell. Now it's not limited and that's problem for my jail system I'm creating.

If the cell will be full, after type /jail command (and another attributs) would number of this cell be hidden. If somebody will entered the command with cell ID, that's full, the error message will be triggered.

Thanks for everything better ideas,
Deimosss

**Comment by mrfloris** [comment]
it would be nice if cmi jail -unless it already does this- would auto flow to the next cell, just starting at one. I don't care which cell they're in for example. So why define it at all? if the cell is empty, put the user in, if it's full, try another one. If there's no other, add to the first one again. And yes, here is where this request comes in place.. if it's at "5 max" then try the other one. if they all file, add anyway ..
